src/main/java/se/knowit/bookitevent/service/map/EventServiceMapImpl.java

```diff
@@ -1,43 +1,60 @@
-package se.knowit.bookitevent.service.map;
-
-import org.springframework.context.annotation.Profile;
-import org.springframework.stereotype.Service;
-import se.knowit.bookitevent.model.Event;
-import se.knowit.bookitevent.service.EventService;
-
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.NoSuchElementException;
-
-@Service
```

Comment:
I prefer to have all beans that are not Spring specific in one configuration file annotated with @Configuration, so that one class handles the dependency injection/configuration part and the domain specific classes handles the actual business logic see 2 and 3 here https://www.endoflineblog.com/spring-best-practices.